SIAB/spillage/lcao_wfc_analysis.py

```
@@ -83,9 +212,154 @@ def test_wll_gamma(self):
+
+        wfc = read_wfc_lcao_txt(outdir + 'WFC_NAO_GAMMA1.txt')[0]
+        S = read_triu(outdir + 'data-0-S')
+        dat = read_running_scf_log(outdir + 'running_scf.log')
+
+        sigma = _svdlz(wfc, S, 'all', 
+                       dat['natom'], 
+                       dat['nzeta'], 
+                       'max',
+                       'decompose')
+        self.assertEqual(len(sigma), len(dat['natom'])) # number of atom types
+        for i, (nt, nz) in enumerate(zip(dat['natom'], dat['nzeta'])):
```

Ruff

Loop control variable `nt` not used within loop body

Rename unused `nt` to `_nt`

(B007)